test/nodetool/rest_api_mock.py

```diff
@@ -19,6 +19,19 @@
 logger = logging.getLogger(__name__)
 
 
+class approximate_value:
```

Comment:
i think pytest already provides [pytest.approx](https://docs.pytest.org/en/latest/reference/reference.html#pytest.approx). does it offer us what we need here?

Reply:
This doesn't work for me, I have to send this type over the wire.